subjects/graphql/README.md

```diff
@@ -0,0 +1,185 @@
+## GraphQL
+
+### Objectives
+
+The objective of this project is to learn graphql query language by creating profiles. It will be provided,\
+by the platform, a graphql endpoint that is connected to the database. So you can query this endpoint to obtain the information you desire.\
+**Note** that for security reasons some tables are private and some are public, you will only be provided with certain content.
+
+Your profile must have at least:
+
+- Basic user identification
+- XP amount
```

Comment:
not sure I want to force students to show their level / xp, maybe we can let them be more free about what they want to display on this profile ?

what do you think ?

Reply:
yes its a good idea, but the main reason i added this was for the audit questions, that way the questions can be more functional (Is the amount of XP for this student displayed on the profile?).

I can say that they must display on their profile at least 4 sections of content at their choice??